.github/workflows/ci.yaml

```diff
@@ -0,0 +1,38 @@
+name: CI
+on:
+  push:
+    branches: [main]
+    paths:
```

Comment:
Why you specify paths by hand? We don't pay for CI hours, we can run them always